lib/Mojolicious/Plugin/OpenAPI/SpecRenderer.pm

```diff
@@ -85,6 +85,10 @@ sub _render_spec {
   local $spec->{basePath} = $c->url_for($spec->{basePath});
   local $spec->{host}     = $c->req->url->to_abs->host_port;
 
+  if ($self->validator->version ge '3') {
+    delete $spec->{$_} for qw/basePath host/;
+  }
```

Comment:
Please do this instead for less lines of code:

```
delete @$spec{qw(basePath host)} if $self->validator->version ge '3';
```

Reply:
This is now fixed